src/components/menu/MenuHeader.scss

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/variables';
+
+.bdl-MenuHeader {
+    position: absolute;
+    top: 0;
+    right: 0;
+    left: 0;
```

Comment:
Similar to the above. Do we need to use both `right` and `left`?

Reply:
No, but I think I left both in there on accident

Reply:
Oops, I meant to say I do need both because the element does not fully span the full width. There is the option of adding `width: 100%`, but I think that comes with other issues.